### PR TITLE
[Pool Simulator] Add FX form interface

### PR DESCRIPTION
**Pull request by yvesfracari**
https://www.loom.com/share/0b024410007745b69c4894c86321b8f4

### CODE REVIEW

File: apps/balancer-tools/src/components/SearchPoolForm.tsx [hunk 1]
```diff
@@ -172,7 +172,12 @@ export function SearchPoolForm({
                       }}
                     >
                       <div className="flex w-full flex-col items-start">
-                        <span>{pool.symbol}</span>
+                        <span>
+                          {poolTypeFilter?.[0] == "FX" &&
+                          poolTypeFilter?.length == 1
+                            ? pool.name
+                            : pool.symbol}
+                        </span>
```
Comment (ribeirojose) [inline apps/balancer-tools/src/components/SearchPoolForm.tsx:180]: can we add a comment explaining why to do this here?

File: apps/balancer-tools/src/lib/schema.ts [hunk 2]
```diff
@@ -237,11 +231,40 @@ export const Gyro2SimulatorDataSchema = BasePoolSchema.extend({
...
+      const feeAtHalt = (data.delta * (2 - (data.alpha - data.beta))) / 100;
+      return feeAtHalt < 50;
+    },
+    {
+      message:
+        "Fee at halt must be less than 50%. Fee at halt is calculated as (delta * (2 - (alpha - beta)))",
+    },
+  );
+
+// TODO: Rename this to StableSwapDataSchema when pool simulator is finished
```
Comment (ribeirojose) [inline apps/balancer-tools/src/lib/schema.ts:267]: task ref?

File: apps/balancer-tools/src/lib/schema.ts [hunk 3]
```diff
@@ -224,7 +218,7 @@ export const GyroESimulatorDataSchema = BasePoolSchema.extend({
     },
   );
 
-export const Gyro2SimulatorDataSchema = BasePoolSchema.extend({
+export const Gyro2DataSchema = BasePoolSchema.extend({
```
Comment (ribeirojose) [inline apps/balancer-tools/src/lib/schema.ts:221]:
```suggestion
export const Gyro2Schema = BasePoolSchema.extend({
```

**Review by ribeirojose — APPROVED** [review]
👍

File: apps/balancer-tools/src/components/SearchPoolForm.tsx (hunk 1 above)
Comment (yvesfracari) [reply apps/balancer-tools/src/components/SearchPoolForm.tsx:180]: Yes. It is bc FX pools have "BPT" as the symbol for all of them.

Comment (yvesfracari) [reply apps/balancer-tools/src/components/SearchPoolForm.tsx:180]: image

**Pull request merged**